packages/wonder-blocks-form/src/components/checkbox-core.tsx

```diff
@@ -6,17 +6,35 @@ import {addStyle} from "@khanacademy/wonder-blocks-core";
 import Icon from "@khanacademy/wonder-blocks-icon";
 
 import type {IconAsset} from "@khanacademy/wonder-blocks-icon";
-
 import type {ChoiceCoreProps} from "../util/types";
 
+type Checked = boolean | null | undefined;
```

Comment:
**suggestion:** this type can be exported so it can be reused in `checkbox.tsx`. It help us to centralize these types in case there's a future change:

```suggestion
export type Checked = boolean | null | undefined;
```

Reply:
Also, it would be really helpful adding a comment about what `null | undefined` means in this context (or more specifically that this intends to be used when the checkbox is in a indeterminate state.